Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -84,9 +86,9 @@ def self.relocation_formulae
   def self.bottle_dependencies
     @bottle_dependencies ||= begin
       formulae = relocation_formulae
-      gcc = Formula["gcc"]
+      gcc = Formula[preferred_gcc]
```

Comment:
```suggestion
      gcc = Formulary.factory(preferred_gcc)
```
… for consistency with the rest of the Homebrew/brew code base.